.github/rd-release-config.yml

```diff
@@ -94,7 +94,7 @@ autolabeler:
   - label: 'docker'
     files:
       - 'docker/**/*'
-  - label: 'backport release'
+  - label: 'backport stable'
```

Comment:
Somewhat unrelated, but should the [pre-commit ci] PRs also be backported? 

https://github.com/helmholtz-analytics/heat/pull/1781

Reply:
While we're off-topic (really sorry about this), basically every PR that modifies the tests gets labeled `testing`, that's pretty much every PR. Would it make sense to only label `testing` if the title contains `test` or something? Thanks a lot, I will stop now.